Makefile

```diff
@@ -48,6 +48,12 @@ build/_output/stratum.so.1.0.0: modelplugin/Stratum-1.0.0/modelmain.go modelplug
 build/_output/stratum-debug.so.1.0.0: modelplugin/Stratum-1.0.0/modelmain.go modelplugin/Stratum-1.0.0/stratum_1_0_0/generated.go
 	CGO_ENABLED=1 go build -o build/_output/debug/stratum-debug.so.1.0.0 -gcflags "all=-N -l" -buildmode=plugin -tags=modelplugin ./modelplugin/Stratum-1.0.0
 
+build/_output/junos.so.19.3R1.8: modelplugin/Junos-19.3R1.8/modelmain.go modelplugin/Junos-19.3R1.8/junos_19_3R1_8/generated.go
+	CGO_ENABLED=1 go build -o build/_output/junos.so.19.3R1.8 -buildmode=plugin -tags=modelplugin ./modelplugin/Junos-19.3R1.8
+
+build/_output/junos-debug.so.19.3R1.8: modelplugin/junos-19.3R1.8/modelmain.go modelplugin/junos-19.3R1.8/junos_19_3R1_8/generated.go
```

Comment:
I think you can leave out the "debug" target - we used to use a debug version of onos-config at one stage, but now I don't think we can use it at all.

Reply:
@SeanCondon ok, I can either remove it in this request or do a follow-up separately  to remove all 5 `*debug.so*` files in a single go (which would avoid having inconsistencies across the plugins for a while). WDYT ?

Reply:
I'd say just remove this JunOS one - best not to add it in. I will remove the other 4 when I move to own repo for plugins